src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -379,102 +306,24 @@ public void Execute_OnlyFailuresInPOI_SavesFile_ReturnsComplete_FailuresOnly()
-
-                AssertCompleteResult(result, 0, 0, 0, 2);
-            }
-        }
-
-        [TestMethod]
-        [Timeout(2000)]
-        public void Execute_MixedResultsInPOI_SavesFile_ReturnsComplete_CorrectMixedResults()
-        {
-            using (ShimsContext.Create())
-            {
-                ShimSelectAction selectAction = new ShimSelectAction()
+                try
                 {
-                    SetCandidateElementA11yElement = (element) => { },
-                    Select = () => true,
-                    POIElementContextGet = () => new ElementContext(
-                        CreateA11yElement(
-                            new List<A11yElement>
-                            {
-                                CreateA11yElement(new List<A11yElement>(), new List<ScanStatus>
-                                {
-                                    ScanStatus.Fail, ScanStatus.Pass   // Will count as failure
-                                }),
-                                CreateA11yElement(new List<A11yElement>(), new List<ScanStatus>
-                                {
-                                    ScanStatus.Pass  // Will count as pass
-                                }),
-                                CreateA11yElement(new List<A11yElement>(), new List<ScanStatus>
-                                {
-                                    ScanStatus.Uncertain, ScanStatus.Uncertain, ScanStatus.ScanNotSupported  // Will count as unsupported
-                                }),
-                                CreateA11yElement(new List<A11yElement>(), new List<ScanStatus>
-                                {
-                                    ScanStatus.Pass, ScanStatus.Uncertain, ScanStatus.NoResult  // Will count as uncertain
-                                }),
-                                CreateA11yElement(new List<A11yElement>(), new List<ScanStatus>
-                                {
-                                    ScanStatus.Uncertain, ScanStatus.NoResult  // Will count as uncertain
-                                }),
-                            },
-                            new List<ScanStatus>
-                            {
-                                ScanStatus.Pass  // Will count as pass
-                            }))
-                };
-
-                InitializeShims(shimTargetElementLocator: true,
-                    selectAction: selectAction, elementBoundExceeded: false, shimUiFramework: true,
-                    setTestModeSucceeds: true, shimScreenCapture: true, shimSnapshot: true, shimSarif: true);
-
-                var config = Config.Builder.ForProcessId(-1).Build();
-
-                SnapshotCommandResult result = SnapshotCommand.Execute(config, OutputFileHelperStub);
+                    results = SnapshotCommand.Execute(config, OutputFileHelperStub, resultsAssembler);
+                }
+                catch (System.Exception ex)
```

Comment:
"System." is not needed

```suggestion
                catch (Exception ex)
```

Reply:
You're correct. This should not be here. I will get rid of it when I rework the snapshot unit tests to remove Fakes. That change is very soon in coming.

Reply:
Your right. This was only for debugging. I will take it out when I rework the snapshot command unit tests to remove Fakes.